packages/apps/human-app/frontend/src/shared/utils/time.ts

```diff
@@ -0,0 +1,22 @@
+export type ProposalStatus = 'pending' | 'active';
+
+export function getProposalStatus(
+  voteStartMs: number,
+  voteEndMs: number,
+  now: number = Date.now()
+): ProposalStatus {
+  if (voteStartMs <= now && now < voteEndMs) return 'active';
+  return 'pending';
+}
+
+export function formatCountdown(diffMs: number): string {
```

Comment:
If you pass `targetDate` (as a `Date` or `ms`) - the called won't need to get `now` every time, so usage of this util would be cleaner